### DISCUSSION TITLE
Community Slack signup link is broken

### DISCUSSION THREAD

**Discussion by ryanmerolle** [StackStorm/st2#6169] [General]
The community Slack signup link is broken. It’s posted on Twitter, the website, github, etc.

 https://stackstorm.com/community-signup

![image](https://github.com/StackStorm/st2/assets/9010275/e17040ba-c8d9-432f-90ba-e7ed1e6e2599)

**Comment by nzlosh** [comment]
Thank you for pointing this out, a new invitation should be available on the website in the next day or two.

**Reply by ryanmerolle** [reply]
Can you at least share the link here please?

**Reply by nzlosh** [reply]
My apologies, the link is now live on the website.
